gocourse06/zoo.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const openRequest = "open"
+const closeRequest = "close"
```

Comment:
У нас реквест може мати два значення, можна спростити:
```suggestion
type request int

const (
	openRequest  request = iota
	closeRequest
)
```